src/assets/MoonIcon.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+export default function MoonIcon({
```

Comment:
there is a moon icon in chakra why don't we use that one :) check your icons fot existence in the chakra/icons

Reply:
changed it